spec/createsend_rails/deliverer_spec.rb

```diff
@@ -24,6 +31,26 @@
       end
     end
 
+
+    context 'sailthru delivery' do
+      let(:request) { { api: 'abcdef', delivery_system: 'sailthru' } }
+      let(:raw_source) { '{ "smart_email_id": "example-id-555", "delivery_system": "sailthru" }' }
+      let(:message) { double ( { body: { raw_source: { delivery_system: 'sailthru' } } } ) }
+
+      it 'return successfully' do
+        sailthru_mock_objct = double
+        expect_any_instance_of(described_class).to receive(:sailthru_client).and_return(sailthru_mock_objct)
+        expect(sailthru_mock_objct).to receive(:api_post).and_return(true)
+        expect(message).to receive(:to).and_return(['example.email@example.com'])
+        expect(message).to receive(:cc).at_least(:once).and_return(['example_cc_email@example.com'])
+        expect(message).to receive(:bcc).at_least(:once).and_return(['example_bcc_email@example.com'])
+        expect(message.body).to receive(:raw_source).at_least(:once).and_return(raw_source)
```

Comment:
This looks like testing implementation details (e.g. we do not care that the `to` method was called, but rather that its result is used correctly in some other call). So I would add stubs for these methods to the double set up previously.